camelot/parsers/stream.py

```diff
@@ -65,20 +65,20 @@ def __init__(
         flag_size=False,
         strip_text="",
         edge_tol=50,
-        row_tol=2,
-        column_tol=0,
+        row_tol=None,
```

Comment:
I would propose keeping any modifications backwards compatible with use of the current interface, so that we can switch out the old camelot and not break any instances where we are using a single call per table. So I would extend the allowed parameters for row_tol and column_tol from int to int & list. If int, it uses only that value for all tables, only fail if a list is passed that doesn't match the length of the table_areas list.

Reply:
you're right. We could also add a new parameter and deprecate the single one, so we can gradually fix the current usage